govcd/nsxt_distributed_firewall.go

```diff
@@ -0,0 +1,88 @@
+package govcd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+// DistributedFirewall contains a types.DistributedFirewallRules which handles Distributed Firewall
+// rules in a VDC Group
+type DistributedFirewall struct {
+	DistributedFirewallRuleContainer *types.DistributedFirewallRules
+	client                           *Client
+	VdcGroup                         *VdcGroup
+}
+
+// GetDistributedFirewall retrieves Distributed Firewall in a VDC Group which contains all rules
+func (vdcGroup *VdcGroup) GetDistributedFirewall() (*DistributedFirewall, error) {
+	client := vdcGroup.client
+	endpoint := types.OpenApiPathVersion1_0_0 + types.OpenApiEndpointVdcGroupsDfwRules
+	apiVersion, err := client.getOpenApiHighestElevatedVersion(endpoint)
+	if err != nil {
+		return nil, err
+	}
+
+	urlRef, err := client.OpenApiBuildEndpoint(fmt.Sprintf(endpoint, vdcGroup.VdcGroup.Id, "default"))
```

Comment:
Do you think that we should mention somewhere that only `default` FW policy is supported at the moment, or is it too obvious?